fastpair/rust/demo/lib/main.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:flutter/material.dart';
+import 'package:demo/rust.dart';
+
+void main() {
+  api.init();
+  runApp(const FastPairApp());
+}
+
+class FastPairApp extends StatelessWidget {
+  const FastPairApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Fast Pair',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: const HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatelessWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Fast Pair"),
+      ),
+      body: Center(
+        child: StreamBuilder(
+          // Retrieve device info stream from Rust side.
+          stream: api.eventStream(),
+          builder: (context, deviceInfo) {
+            if (deviceInfo.hasData) {
+              return Column(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    // `deviceInfo.data[0]` holds device name.
+                    // `deviceInfo.data[1]` holds image URL.
```

Comment:
Can we make these variables? This can help with type safety, would remove the need to explain what these array entries represent and make it easier to update if `data` ever changes